css/home.css

```diff
@@ -506,11 +527,18 @@ Hero Section
 	}
 
 	/* overriding global styles */
-	.hero h3,
+	.hero h1,
 	.hero p {
 		color: var(--color-white);
 	}
 
+	.hero h1 {
+		font-family: "Josefin Sans";
+		font-size: 56px;
+		font-weight: 700;
+		line-height: 58px;
+	}
+
 	/* ================================
 Community Partnerships Section
 ===================================  */
```

Comment:
<img width="534" alt="Screen Shot 2024-01-26 at 8 30 15 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/5c02db4c-b763-4b07-9b40-9a30c91401a1">


I think I found the issue with the red Community Partnerships section on mobile: 

1. In the HTML, The `wrapper` class is on the section but it should be on the div inside. 
Before: 
```
<section class="community-partnerships wrapper">
   <!-- "community" contains the image and the "partnerships-paragraph" -->
	  <div class="community align-text">
```

After: 
```
<section class="community-partnerships">
   <!-- "community" contains the image and the "partnerships-paragraph" -->
	  <div class="community wrapper align-text">
```

2. Then move the background color to the `.community-partnerships` class, instead of `.community` (line 65)

3. Delete this as the wrapper should work now (line 75)
```
/* overriding .wrapper class */
.community-partnerships.wrapper {
	width: 100%;
}
```

Reply:
I did what you requested and this happened in desktop
![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/77703524/1d08b61e-4436-4969-8ebe-8d0fdfc6e714)

I have a different solution - we can discuss

Reply:
resolved issue